sonar-project.properties

```diff
@@ -1,4 +1,4 @@
-sonar.projectKey=open-cluster-management_kubernetes-dependency-watches
```

Comment:
Can we have them use a different key instead? Using `stolostron_` as a key differs from all the other repos, and I don't want to start using a different key all of a sudden for this one repo.

Reply:
It's not a big deal on the surface level, but as soon as someone tries to do API queries (like CICD or Security), this would become a annoying exception.